src/command_modules/azure-cli-devops/azure/cli/command_modules/devops/_params.py

```diff
@@ -0,0 +1,67 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import CliArgumentType, register_cli_argument
+from azure.cli.core.commands.parameters import (
+    name_type,
+    resource_group_name_type, 
+    get_resource_group_completion_list)
+
+registry_name = CliArgumentType(
+    options_list=('--registry-name', '-r'),
+    help='Docker registry name', 
+    required = False
```

Comment:
You can remove the `required` parameters that you've set here.
The CLI infrastructure will inspect the method signature to determine if a parameter is required or not.

So in this case, it will inspect `add_release()`. If a parameter really is not required, the method signature should have a default set for that parameter.